archie-utils/src/main/java/com/nedap/archie/json/JacksonUtil.java

```diff
@@ -41,20 +49,18 @@
 public class JacksonUtil {
 
     //threadsafe, can be cached
-    private static final ConcurrentHashMap<RMJacksonConfiguration, ObjectMapper> objectMapperByConfiguration = new ConcurrentHashMap<>();
-
-    private static final String DEFAULT_TYPE_PARAMETER = "@type";
+    private static final ConcurrentHashMap<ArchieJacksonConfiguration, ObjectMapper> objectMapperByConfiguration = new ConcurrentHashMap<>();
 
     /**
      * Get an object mapper that works with Archie RM and AOM objects. It will be cached in a static variable for
-     * performance reasons
-     * @return
+     * performance reasons. returns a standards compliant version of the object mapper.
+     * @return the requested object mapper.
      */
     public static ObjectMapper getObjectMapper() {
-        return getObjectMapper(new RMJacksonConfiguration());
+        return getObjectMapper(ArchieJacksonConfiguration.createStandardsCompliant());
```

Comment:
it is in the issue description as a breaking change, so will be added to the release notes.

Reply:
It is in the issue description as a breaking change. It will be added to the release notes, as I will check all merges pull requests when writing release notes.